### PR TITLE
python.pkgs.kafka-python: init at 1.3.5

### PR DESCRIPTION
**Pull request by themoritz**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/kafka-python/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  name = "${pname}-${version}";
+  version = "1.3.5";
+  pname = "kafka-python";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "19m9fdckxqngrgh0www7g8rgi7z0kq13wkhcqy1r8aa4sxad0f5j";
+  };
+
+  buildInputs = [ tox ];
```
Comment (FRidh) [inline pkgs/development/python-modules/kafka-python/default.nix]: checkInputs

File: pkgs/development/python-modules/kafka-python/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  meta = with stdenv.lib; {
+    description = "Pure Python client for Apache Kafka";
+    homepage = https://github.com/dpkp/kafka-python;
+    license = licenses.asl20;
+    maintainers = with maintainers; [ ];
```
Comment (FRidh) [inline pkgs/development/python-modules/kafka-python/default.nix]: Will you maintain this?

Comment (themoritz) [reply pkgs/development/python-modules/kafka-python/default.nix]: I can try, but there will be no guarantee that I'll do it reliably :)

File: pkgs/development/python-modules/kafka-python/default.nix (hunk 1 above)
Comment (themoritz) [reply pkgs/development/python-modules/kafka-python/default.nix]: If tests run successfully without any `checkInputs`, is there a need to add them?

File: pkgs/development/python-modules/kafka-python/default.nix (hunk 2 above)
Comment (FRidh) [reply pkgs/development/python-modules/kafka-python/default.nix]: If you can add an expression you can maintain it. It basically means you become responsible for the functioning. In principle we shouldn't have any (new) expressions without maintainers.

**Comment by FRidh** [comment]
Pushed 80cc3de45a3 to master.

File: pkgs/development/python-modules/kafka-python/default.nix (hunk 2 above)
Comment (themoritz) [reply pkgs/development/python-modules/kafka-python/default.nix]: Sure, I can fix it when it breaks.

**Pull request closed without merging**